orchestra/models/__init__.py

```diff
@@ -14,4 +14,5 @@
 from orchestra.models.core.models import Iteration  # noqa
 from orchestra.models.core.models import TimeEntry  # noqa
 from orchestra.models.core.models import TaskTimer  # noqa
+from orchestra.models.core.models import Todo  # noqa
```

Comment:
I feel like you complained about this to me :-)

If you leave this here, the way to get rid of the noqas is to define `__all__`, which makes some sense if the point of this code is to gather a bunch of symbols in one module. (Though python people fight about it a lot).

Reply:
did __all___

overall i dislike this model of doing things, but refactors/cleanups on this PR are out of the question :)

Reply:
Nit: but now you should be able to remove all the `# noqa`s.

Reply:
ok